src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-controller.php

```diff
@@ -522,11 +522,10 @@ public function get_theme_item_permissions_check( $request ) {
 	 * @since 5.9.0
 	 *
 	 * @param WP_REST_Request $request The request instance.
-	 *
 	 * @return WP_REST_Response|WP_Error
 	 */
 	public function get_theme_item( $request ) {
-		if ( wp_get_theme()->get_stylesheet() !== $request['stylesheet'] ) {
+		if ( wp_get_theme()->get_stylesheet() !== str_replace( '%20', ' ', $request['stylesheet'] ) ) {
```

Comment:
We should implement a sanitize callback for stylesheet

Reply:
Yup, you're right. Thanks @spacedmonkey. Done in https://github.com/WordPress/wordpress-develop/pull/2034/commits/03bf811e413e6e76d4a864dce0cda339dd930346.